pkg/ebpf/c/tracee.bpf.c

```diff
@@ -2732,6 +2834,22 @@ int BPF_KPROBE(trace_do_exit)
     return events_perf_submit(&data, DO_EXIT, code);
 }
 
+SEC("kprobe/security_file_ioctl")
+int BPF_KPROBE(trace_tracee_trigger_event )
+{
+    event_data_t data = {};
+
+    if (!init_event_data(&data, ctx))
+        return 0;
+
+    unsigned int cmd = PT_REGS_PARM2(ctx);
+
+    if (cmd == IOCTL_FETCH_SYSCALLS && get_config(CONFIG_TRACEE_PID) == data.context.host_pid){
```

Comment:
I don't think so, because we submit an event only if it's driven by Tracee and its IOCTL command.

This means that only Tracee can submit this event in a specific scenario.